### PR TITLE
Add Enchilada page

### PR DESCRIPTION
**Pull request by spideyIII**

### CODE REVIEW

**Review by bananabrann — CHANGES_REQUESTED** [review]
Woohoo first PR! 🥳🥂

Your code and styling looks great. Let's just...
1. Run the Prettier script (see README)
2. Add an `a` tag to the appropriate section on the homepage so users can get to your enchilada recipe

File: src/routes/+page.svelte [hunk 1]
```diff
@@ -10,4 +10,6 @@
 <i>Stay tuned for more recipes!</i>
```
Comment (Chubbar5) [inline src/routes/+page.svelte:10]: Looks great! You're so handsome!

File: src/routes/mexican/enchiladas/+page.svelte [hunk 2]
```diff
@@ -0,0 +1,41 @@
+<div>
+  <h1>Cheese Enchiladas</h1>
+  <img
+    src="https://files.brannan.cloud/recipes/enchiladas.png"
+    alt="A yummy, but definitely cheesey enchilada."
+  />
```
Comment (Chubbar5) [inline src/routes/mexican/enchiladas/+page.svelte:6]: Wow what a great picture!

**Review by Chubbar5 — APPROVED** [review]
Everything looks great, approved!

**Review by bananabrann — APPROVED** [review]

**Pull request merged**